Bixal/move.mil
# fixed react_tools security vulns

Thanks for submitting a pull request! Below are a few things you can do to help us more quickly review your changes.

## Checklist

I have…

      against my changes

## Summary of Changes
Updated modules with react_tools to fix security vulnerabilities.

This pull request…

- Updated several packages.
- Changed code to reflect changes with the packages.

## Testing

To verify the changes proposed in this pull request…

1. Go to the web/modules/custom/react_tools/tools directory.
2.
 ```bash
npm install
```
No vulnerabilities should be listed.
3.  Confirm with:
```bash
npm audit
```
4. Nothing should act different about the website. These are security fixes, not functionality changes.
